core/src/test/java/com/mongodb/stitch/core/testutil/Constants.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.stitch.core.testutil;
+
+public final class Constants {
+  public static final Long DEFAULT_TRANSPORT_TIMEOUT_MILLISECONDS = 15000L;
+}
```

Comment:
[nit] missing newline. Also I'm not a fan of a Constants file, but it seems fine here.